Header header
bool moving
bool goal_reached

aero_control/arm_error error